test/pgtest/range.pt

```diff
@@ -91,112 +91,112 @@ ReadyForQuery
 ParseComplete
 BindComplete
 DataRow {"fields":["empty","{empty}"]}
-DataRow {"fields":["(,1)","{empty,\"(,1)\",\"[-99,-50)\",\"[-1,1)\"}"]}
-DataRow {"fields":["(,2)","{empty,\"(,1)\",\"[-99,-50)\",\"[-1,1)\"}"]}
-DataRow {"fields":["(,)","{empty,\"(,1)\",\"(,)\",\"[-99,-50)\",\"[-1,1)\",\"[0,)\",\"[50,99)\"}"]}
-DataRow {"fields":["[-1,1)","{empty,\"[-1,1)\"}"]}
-DataRow {"fields":["[-1,2)","{empty,\"[-1,1)\"}"]}
+DataRow {"fields":["(,1)","{empty,(,1),[-99,-50),[-1,1)}"]}
```

Comment:
It'd be real nice to also do a query in binary mode so we can verify the bytes are identical too. We don't do that for anything else afaik, but if you're at all worried about our encoding correctness, that'd be the easiest way to check it.

Reply:
Oh nice––you have a doc for executing queries in binary mode?

Reply:
https://github.com/MaterializeInc/materialize/blob/50b2d9ee219599b14ebcebad8a29cdd311c3fcb6/test/pgtest/cursors.pt#L521 is an example of how to request it, you can see the DataRow with bytes below.

Reply:
ty!